torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -191,6 +199,20 @@ bool isSupported(Node* node) {
       }
     }
 
+    // Value is only supported if operands are floats.
+    if (node->isMemberOf(float_only_operator_set)) {
+      for (const Value* v : node->inputs()) {
+        if (auto const& tt = v->type()->cast<TensorType>()) {
+          auto const& st = tt->scalarType();
+          if (!st || !isFloatingType(*st)) {
+            return false;
+          }
+        } else if (!v->type()->cast<FloatType>()) {
```

Comment:
This doesn't include Half and Double, does it?

Reply:
It might. We don't seem to differentiate statically halfs, doubles and single-points, @eellison ?

Reply:
this check is on a non-Tensor type (Scalar value), so Half/Double doesn't apply here